nvaccess/nvda
# Show reset config as a menu item in secure mode

### Link to issue number:
None

### Summary of the issue:
Currently a user can temporarily reset to factory defaults in secure mode by pressing the default input gesture `NVDA+control+r` 3 times.
A user using NVDA in a shared kiosk may find this command helpful to temporarily use NVDA with factory settings, so that they can log in and then use their own settings profile.

This is safe, as resetting to factory defaults doesn't save the settings profile to disk. Users can revert to the saved configuration for secure screens by:
- using the menu
- `NVDA+control+r`
- restarting NVDA

The feature "reset configuration to factory defaults" is hidden from the NVDA menu in secure mode, even though it is exposed through a default input gesture.

### Description of how this pull request fixes the issue:

Show "reset configuration to factory defaults" as a menu item in secure mode

### Testing strategy:

Manual testing:
 - Change a preference in NVDA preferences 
 - Copy your configuration to secure screens
 - From a secure screen
   - Note that the menu item is visible
   - Close the preference dialog
   - Reset configuration to factory defaults using the menu item
   - Confirm that the changed preference has been reset to the factory default
 - Restart the machine
 - on the sign-in screen, confirm that the preference has been restored to the saved config - not the factory default

### Known issues with pull request:

None

### Change log entries:
Bug fixes

```t2t
- "Reset configuration to factory defaults" is now accessible in the NVDA menu during secure mode.
```

### Code Review Checklist:



  - description is up to date
  - change log entries
  - Unit tests
  - System (end to end) tests
  - Manual testing
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
